src/platform/device.gni

```diff
@@ -58,6 +58,7 @@ declare_args() {
 
 declare_args() {
   chip_openthread_ftd = chip_enable_openthread
+  chip_openthread_border_router = 0
```

Comment:
GN supports booleans. This seems like it should be false to become a bool instead of numeric.